kaniko/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: help
+SHELL            := /bin/bash
+DOCKER_REPO_NAME := kaniko
+
+help:
+	@echo 'Available Commands:'
+	@egrep '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":"}; { if ($$3 == "") { printf " - \033[36m%-18s\033[0m %s\n", $$1, $$2 } else { printf " - \033[36m%-18s\033[0m %s\n", $$2, $$3 }}'
+
+#==============================================================#
+# DOCKER                                                       #
+#==============================================================#
+build: ## build docker image
+	docker build -t public.ecr.aws/n0c8p2u1/kaniko:latest --build-arg KANIKO_VERSION='1.9.0' .
```

Comment:
Notice we are pushing this to our public ECR repo which can be accessed from here: https://gallery.ecr.aws/n0c8p2u1/

Reply:
FYI @juanmatias just took care of making Kaniko part of our Docker Hub hosted images. Now, we need to make a note to switch to that new image which we are using on a client.
+ @fgauchat-binbash  ☝️ 🙏